docs/guides/audio-config.md

```diff
@@ -4,6 +4,19 @@ This page explains how to get the config files for using the T2 audio device, wh
 
 Before you proceed, make sure you already have `apple_bce` loaded by running `lsmod | grep apple_bce`. If not, follow the instructions on [how to setup the BCE module](https://wiki.t2linux.org/guides/dkms/#installing-modules).
 
+# Enable Pass-Through Kernel Parameters
+
+Cat `cat /proc/cmdline` and ensure that your kernel parameters contain `intel_iommu=on iommu=pt pcie_ports=compat`.
+
+If not present, you'll have to update your bootup kernel params:
+
+- edit `/etc/default/grub` and update `GRUB_CMDLINE_LINUX` to include `intel_iommu=on iommu=pt pcie_ports=compat`
+- Apply your edits by running `sudo update-grub` on ubuntu or `grub-mkconfig -o /boot/grub/grub.cfg` for other distros
+- Reboot and ensure `cat /proc/cmdline` contains those params
+
+NOTE: if you use systemd-boot, you'll instead edit your boot conf files to add `intel_iommu=on iommu=pt pcie_ports=compat` to the options line.
+- The location of these conf files can vary by distro. Arch stores them in `/boot/efi/loader/entries/arch.conf` and `/boot/efi/loader/entries/arch-fallback.conf`
```

Comment:
The files they have to edit will have the ".conf" extension and be in the `loader/entries/` folder on their EFI partition. This will most likely be `/boot/efi/loader/entries`.

Those example arch locations are actually just for if you used archinstall, if arch is manually installed the user could give them a different name, so they probably shouldn't be here (or in the arch guide ;-;)

Reply:
Thanks. I lifted your verbage into the note